robotbase/src/main/java/frc/robot/Constants.java

```diff
@@ -131,7 +133,10 @@ public static class PHEONIX_TUNER {
             SWERVE.INVERT_RIGHT_SIDE);
 
     public static final SwerveDrivetrainConstants DRIVETRAIN_CONSTANTS =
-        new SwerveDrivetrainConstants().withPigeon2Id(CAN_ID.PIGEON_ID).withCANbusName("");
+        new SwerveDrivetrainConstants()
+            .withPigeon2Id(CAN_ID.PIGEON_ID)
```

Comment:
Why are we making 2 pigeons?

Reply:
i removed the other .withPigeon2Id, so we arent telling it twice